website/docs/cloud-docs/workspaces/health.mdx

```diff
@@ -60,6 +60,13 @@ Terraform Cloud pauses health assessments if the latest run ended in an errored
 
 Health assessments may take longer to complete when you enable health assessments in many workspaces at once or your workspace contains a complex configuration with many resources.
 
+### On Demand Assessments
+
+If you are an administrator for the workspace and all [workspace requirements](/terraform/cloud-docs/workspaces/health#workspace-requirements) are satisfied, a **Start health assessment** button is available on the Health pages with Ready status.
```

Comment:
i'd like to rephrase this first sentence a little bit, but a few clarifying questions first -- 
- will the button not be visible at all if the workspace doesnt satisfy requirements, or just wont be clickable?
- what do we mean by "with ready status"?

Reply:
There can be multiple reasons based on the requirements why the button wont be clickable. 
<img width="701" alt="Screenshot 2023-03-09 at 9 36 16 AM" src="https://user-images.githubusercontent.com/18754820/224109795-0633c9b6-c864-4177-aea4-2ddd7c5d724c.png">
These are the different states the button will be in -
<img width="419" alt="Screenshot 2023-03-09 at 9 34 28 AM" src="https://user-images.githubusercontent.com/18754820/224110110-c97dda4c-bc6e-4735-bdfc-753705913977.png">

I was not sure how much of these details we want on the docs especially since this is not a "public" endpoint.